<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1163</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolbar_menu">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionSave"/>
   <addaction name="actionOpen"/>
   <addaction name="actionExport"/>
  </widget>
  <widget class="QToolBar" name="toolbar_graph">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPlot_GraphY"/>
   <addaction name="actionPlot_GraphXY"/>
   <addaction name="actionPlot_CurveXY"/>
   <addaction name="actionPlot_MapXYZ"/>
   <addaction name="actionHistogram"/>
   <addaction name="actionPlot_Cloud_3D"/>
   <addaction name="actionPlot_Cloud_XYZ"/>
   <addaction name="actionFFT"/>
   <addaction name="actionPlot_Gain_Phase"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>icons/open.png</normaloff>icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>icons/save.png</normaloff>icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionPlot_GraphY">
   <property name="icon">
    <iconset>
     <normaloff>icons/graphY.png</normaloff>icons/graphY.png</iconset>
   </property>
   <property name="text">
    <string>Graph Y</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionPlot_GraphXY">
   <property name="icon">
    <iconset>
     <normaloff>icons/graphYX.png</normaloff>icons/graphYX.png</iconset>
   </property>
   <property name="text">
    <string>Graphs XY</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionMulti_Plot_y">
   <property name="text">
    <string>Multi Plot Y</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionRand">
   <property name="text">
    <string>Rand</string>
   </property>
  </action>
  <action name="actionHistogram">
   <property name="icon">
    <iconset>
     <normaloff>icons/histogram.png</normaloff>icons/histogram.png</iconset>
   </property>
   <property name="text">
    <string>Histogram</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionPlot_MapXYZ">
   <property name="icon">
    <iconset>
     <normaloff>icons/mapYX.png</normaloff>icons/mapYX.png</iconset>
   </property>
   <property name="text">
    <string>Map XYZ</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionTile">
   <property name="text">
    <string>Tile</string>
   </property>
   <property name="shortcut">
    <string>Alt+Down</string>
   </property>
  </action>
  <action name="actionCascade">
   <property name="text">
    <string>Cascade</string>
   </property>
   <property name="shortcut">
    <string>Alt+Up</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="text">
    <string>Next</string>
   </property>
   <property name="shortcut">
    <string>Alt+Right</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="text">
    <string>Previous</string>
   </property>
   <property name="shortcut">
    <string>Alt+Left</string>
   </property>
  </action>
  <action name="actionPlot_CurveXY">
   <property name="icon">
    <iconset>
     <normaloff>icons/curveYX.png</normaloff>icons/curveYX.png</iconset>
   </property>
   <property name="text">
    <string>Curve XY</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionPlot_Cloud_3D">
   <property name="icon">
    <iconset>
     <normaloff>icons/cloudZYX.png</normaloff>icons/cloudZYX.png</iconset>
   </property>
   <property name="text">
    <string>Cloud 3D</string>
   </property>
  </action>
  <action name="actionPlot_Surface_3D">
   <property name="text">
    <string>Plot Surface 3D</string>
   </property>
  </action>
  <action name="actionPoints">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Points</string>
   </property>
  </action>
  <action name="actionLines">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Lines</string>
   </property>
  </action>
  <action name="actionPlot_Gain_Phase">
   <property name="icon">
    <iconset>
     <normaloff>icons/bode.png</normaloff>icons/bode.png</iconset>
   </property>
   <property name="text">
    <string>Bode</string>
   </property>
  </action>
  <action name="actionFFT">
   <property name="icon">
    <iconset>
     <normaloff>icons/fourier.png</normaloff>icons/fourier.png</iconset>
   </property>
   <property name="text">
    <string>FFT</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset>
     <normaloff>icons/latex.png</normaloff>icons/latex.png</iconset>
   </property>
   <property name="text">
    <string>Latex</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionPlot_Cloud_XYZ">
   <property name="icon">
    <iconset>
     <normaloff>icons/cloudYX.png</normaloff>icons/cloudYX.png</iconset>
   </property>
   <property name="text">
    <string>Cloud 2D</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>icons/new.png</normaloff>icons/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
